.cloudbees/workflows/workflow.yaml

```diff
@@ -65,6 +65,23 @@ jobs:
         values: |
           service:
             type: ClusterIP
+    - name: upgrade an remote/OCI helm chart
```

Comment:
To properly test an upgrade, on every commit, shouldn't we un-install the charts at the end of the workflow?

Reply:
Since the workflow/test job [creates a new ephemeral k8s Namespace](https://github.com/cloudbees-io/helm-install/blob/3c493f190485846532de36633b13d7b90eae1be4/.cloudbees/workflows/workflow.yaml#L45) for every git commit it tests, we can be sure that the tests start with a clean state. The ephemeral Namespace (including all chart releases that have been installed by the test job) will be cleaned up by our CronJob afterwards.